best-practices/README.md

```diff
@@ -12,8 +12,8 @@ These files aim to outline our best practices when it comes to engineering at Ho
 - [JavaScript](/best-practices/javascript.md)
 - [jQuery](/best-practices/jquery.md)
 - [Markdown](/best-practices/markdown.md)
-- [npm](/best-practices/npm.md)
+- [NPM](/best-practices/npm.md)
```

Comment:
Oh wow, you are right! I went through changing these explicitly because it is an acronym and therefore should be uppercased, but you are right... everyone else on the web writes it in lower case! I will change this, good spot 😄.